README.rst

```diff
@@ -133,6 +133,62 @@ input queries before sending them to the API:
 Run ``python -m autoextract --help`` to get description of all supported
 options.
 
+Errors
+~~~~~~
+
+The following errors could happen while making requests:
```

Comment:
I prefer _fetching_ or _downloading_ instead of _making_ here.
But it's personal

Reply:
IMO

- you fetch responses, not requests
- you download responses, not requests
- you make/send a request and server answers/sends a response